dev/CommonStyles/Deprecated_themeresources.xaml

```diff
@@ -179,6 +188,11 @@
             <StaticResource x:Key="ExpanderDisabledForeground" ResourceKey="TextFillColorDisabledBrush" />
             <StaticResource x:Key="ExpanderDisabledBorderBrush" ResourceKey="TextFillColorDisabledBrush" />
             <Thickness x:Key="ExpanderBorderThickness">2</Thickness>
+
+            <CornerRadius x:Key="ListViewItemCornerRadius">4</CornerRadius>
+            <CornerRadius x:Key="ListViewItemCheckBoxCornerRadius">4</CornerRadius>
+            <CornerRadius x:Key='GridViewItemCornerRadius'>4</CornerRadius>
+            <CornerRadius x:Key='GridViewItemCheckBoxCornerRadius'>4</CornerRadius>
```

Comment:
Previous resource value was 3, why is it 4 now?

Reply:
Same as above ^